frontend/src/metabase/ui/components/inputs/Input/Input.styled.tsx

```diff
@@ -0,0 +1,121 @@
+import { getSize, rem } from "@mantine/core";
+import type { InputStylesParams, MantineThemeOverride } from "@mantine/core";
+
+const SIZES = {
+  xs: rem(28),
+  md: rem(40),
+};
+
+const PADDING = 12;
```

Comment:
We need to do some basic math with these vars, that's why I don't convert them to `rem` here to avoid using `calc` where not needed